t/porting/regen.t

```diff
@@ -111,6 +111,6 @@ foreach (@progs) {
     system $command
         and die <<~"HINT";
```

Comment:
Now we're not interpolating `$_` we should single quote the heredoc.

Reply:
agree

Reply:
Fixed in 1044e25ef2; please re-review.